common/ferrostar/src/routing_adapters/valhalla.rs

```diff
@@ -108,6 +108,8 @@ mod tests {
         horizontal_accuracy: 6.0,
         course_over_ground: None,
         timestamp: SystemTime::UNIX_EPOCH,
+        speed: None,
+            speed_accuracy: None,
```

Comment:
Formatting looks a bit off; I haven't added rustfmt checks to CI, but I should :P Just run `cargo fmt` (from the `common` directory) locally to reformat everything.